codecov.yml

```diff
@@ -1,25 +0,0 @@
-codecov:
```

Comment:
As discussed on MM, remaining python codecov check to be investigated separately